### PR TITLE
Add a rnn model and metric plotting utility

### PR DESCRIPTION
**Pull request by unrahul**
- update the FCN, increase layers and add dropout
- add a GRU based RNN with tfidf embedding
- Add metric plotting function to plot accuracy and error

### CODE REVIEW

File: github-issue-classification/python/train.py [hunk 1]
```diff
@@ -22,9 +22,14 @@
 import os
 
 import pandas as pd
+import matplotlib.pyplot as plt
 import numpy as np
-from tensorflow.keras.models import Sequential
```
Comment (Meadosc) [inline github-issue-classification/python/train.py]: where is the import for model?

File: github-issue-classification/python/train.py [hunk 2]
```diff
@@ -41,20 +46,57 @@
 except KeyError:
     DATADIR = "/workdir/data/tidy/"
 
+
+def _fcn(dim=1000, classes=10, dropout=0.3):
+    "fully connected net."
+    _model = Sequential()
```
Comment (Meadosc) [inline github-issue-classification/python/train.py]: Wasn't sequential removed earlier?

File: github-issue-classification/python/train.py [hunk 3]
```diff
@@ -41,20 +46,57 @@
...
+    if fcn:
+        model = _fcn(dim=1000, classes=10)
+    else:
+        model = _rnn(dim=1000, classes=10)
+    model.compile(loss="binary_crossentropy",
+                  optimizer="adam", metrics=["accuracy"])
+    return model
+
+
+def plot_metrics(history, metric):
```
Comment (DnPlas) [inline github-issue-classification/python/train.py]: Nice addition
